embedding-stores/pgvector/runtime/src/main/java/io/quarkiverse/langchain4j/pgvector/PgVectorAgroalPoolInterceptor.java

```diff
@@ -16,8 +16,9 @@ public class PgVectorAgroalPoolInterceptor implements AgroalPoolInterceptor {
 
     @Override
     public void onConnectionCreate(Connection connection) {
+        ConfigUtils.isProfileActive("dev");
```

Comment:
this seems unnecessary

Reply:
Realized it was a statement I didn't delete (Ask Copilot). Already removed it